### PR TITLE
remove duplicate icons

### PR DESCRIPTION
**Pull request by powell759**

### CODE REVIEW

**Comment by jvoisin** [comment]
Feel free to group more from [1379](https://github.com/AliasIO/Wappalyzer/issues/1379) in this pull request :)

File: src/apps.json [hunk 1]
```diff
@@ -2051,10 +2051,10 @@
 			"cats": [
 				1
 			],
-			"html": "<link[^>]* href=\"\/DovetailWRP\/",
+			"html": "<link[^>]* href=\"\\/DovetailWRP\\/",
```
Comment (jvoisin) [inline src/apps.json]: This should go in a separate pull-request in my opinion.

File: src/apps.json [hunk 2]
```diff
@@ -459,16 +459,16 @@
...
-      "cats": [
-        19
-      ],
-      "headers": {
-        "Server": "AmazonS3"
-      },
-      "icon": "aws-s3.svg",
-      "website": "aws.amazon.com/s3/"
-    },
+		"Amazon S3": {
```
Comment (jvoisin) [inline src/apps.json]: This should be addressed in another pull request.

File: src/apps.json [hunk 3]
```diff
@@ -2745,39 +2745,27 @@
 			},
 			"website": "gitbook.io"
 		},
-		"Gitiles": {
```
Comment (jvoisin) [inline src/apps.json]: This should go in another pull-request.

File: src/apps.json [hunk 4]
```diff
@@ -5194,17 +5194,17 @@
 		},
 		"Nette Framework": {
```
Comment (jvoisin) [inline src/apps.json]: This should go in another pull-request.

File: src/apps.json [hunk 5]
```diff
@@ -9530,22 +9530,6 @@
 			"implies": "PHP",
 			"website": "www.phpdoc.org"
 		},
-		"phpliteadmin": {
```
Comment (jvoisin) [inline src/apps.json]: This should go in another pull-request.

**Comment by jvoisin** [comment]
Your pull-request contains a lot of unrelated modification, it would be great if you could split them into different meaningful pull-requests. If you're too lazy, it's ok, your modifications looks cool to me :)

**Comment by powell759** [comment]
I ran wappalyzer build to make sure it everything was good before commiting and it automatically alphabetized/orderers some elements and added some escapes - I was under the impression this was meant to happen - I can separate them later today if you want ( otherwise I'll just keep that in mind for future pulls)

**Comment by powell759** [comment]
I looked into this. Looks like jsonlint is being used in the wappalyzer validate script. So I think all these artifacts are just left over from additions to apps.json that weren't validated with the script.

**Comment by AliasIO** [comment]
Thanks!
